ipn/ipnserver/server_notwindows.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !windows
+
+package ipnserver
+
+import (
+	"net"
+
+	"inet.af/peercred"
+)
+
+// getConnIdentity returns the localhost TCP connection's identity information
+// (pid, userid, user). If it's not Windows (for now), it returns a nil error
+// and a ConnIdentity with NotWindows set true. It's only an error if we expected
+// to be able to map it and couldn't.
+func (s *Server) getConnIdentity(c net.Conn) (ci connIdentity, err error) {
+	ci = connIdentity{Conn: c, NotWindows: true}
+	ci.NotWindows = true
+	_, ci.IsUnixSock = c.(*net.UnixConn)
+	ci.Creds, _ = peercred.Get(c)
```

Comment:
This could error due to selinux policies or similar, why not return an error in this cases? Field debugging without logging the cause may be extra difficult.

Reply:
Again, this is all code movement.

Reply:
I do not want to modify this behavior without also testing it on all the *nix platforms.